sdk/web-pubsub/web-pubsub-client/src/webPubSubClient.ts

```diff
@@ -560,6 +566,27 @@ export class WebPubSubClient {
     }
   }
 
+  private async _trySendPing(): Promise<void> {
+    // If the ping timeout is not null, it means that the last ping message is still waiting for a pong response message
+    // So we don't need to send a new ping message.
+    if (this._pingTimeout != null) {
```

Comment:
Seems it's not good. Consider you're sending ping every 20 seconds and the timeout is 60 seconds. You're giving a chance that ping may lost 1 and 2 times. But here your `_pingTimeout` prevent sending another ping.